app/views/lecturers/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: @lecturer, local: true) do |form| %>
+  <% if @lecturer.errors.any? %>
```

Comment:
We dont need views for lecturers. the info for lecturers will be presented with the associated resource, subjects